src/components/Modal/Modal.jsx

```diff
@@ -37,7 +38,7 @@ class Modal extends React.Component<Props> {
     return (
       <div {...getDataAttrs(data)} className={style.wrapper}>
         <div className={style.innerWrapper}>
```

Comment:
Why not put it the `innerWrapperStyle` on this wrapper? 
if you need in the card maybe better name would be `innerCardStyle`